pybliometrics/scopus/tests/test_AbstractRetrieval.py

```diff
@@ -25,6 +25,8 @@
 ab8 = AbstractRetrieval("2-s2.0-84951753303", view="REF", refresh=30)
 # Collaboration
 ab9 = AbstractRetrieval("2-s2.0-85097473741", view="FULL", refresh=30)
+# ENTITLED view
+ar_entiteled = AbstractRetrieval('10.1109/Multi-Temp.2019.8866947', view='ENTITLED', refresh=30)
```

Comment:
Even though it's just a minor object around here, please call it `ar_entitled`, i.e., without the typo